### ISSUE TITLE
Add ability to sort out test classes from the view

### ISSUE THREAD

**Issue by radaczynski** [jmhofer/eCobertura#7]
I would like to be able to see the coverage only on application classes (in maven that would be those residing in src/main as opposed to src/test, but perhaps identifying them by name is a better idea? or by other means?)

**Comment by jmhofer** [comment]
Do you know about the filters in the run configuration I added in one of the recent versions? - Do I understand correctly that you want to filter only the view, but still collect coverage data for all classes?

**Comment by radaczynski** [comment]
Nope, I did not know that. Will check them out, thanks! And yes - this is exactly what I would like to achieve.

**Comment by jmhofer** [comment]
So we'd need some kind of filter panel for the coverage view in addition to the run configuration filters...

**Comment by argoyle** [comment]
Would it be possible to add "Default filters" in the plugin configuration that is picked up by new launch configurations? I usually exclude *Test* and *Mock* and a configuration like that would help me much.

**Comment by bradcupit** [comment]
filtering on directory would be even more useful
example: exclude 'test' or '/src/test/java'
